src/mappings/Competition/mapping.ts

```diff
@@ -100,6 +140,31 @@ export function handleNewVote(event: NewVote): void {
       suggestion.save();
     }
     vote.save();
+
+    let competition = Competition.bind(event.address);
+    competitionProposal.winningSuggestions = [];
+    let winningSuggestions = competitionProposal.winningSuggestions;
+    debug(competitionProposal.suggestions.toString() + '');
+    for (let i = 0; i < competitionProposal.suggestions.length; i++) {
+      let suggestions = competitionProposal.suggestions;
+      if (suggestions != null) {
```

Comment:
I don't get why these lines are _in_ the `for` loop; also do not get how `suggestions` could be `null` without the call to `competitionProposa.suggestions.length` throwing an error...